src/vl-radio.js

```diff
@@ -68,7 +68,9 @@ export class VlRadio extends VlElement(HTMLElement) {
 
   _check() {
     const host = this.getRootNode().host;
-    const radios = host.getRootNode().querySelectorAll(`vl-radio[data-vl-name='${this.name}']`);
+    const isSlot = host.assignedSlot != undefined;
```

Comment:
gaat dat niet altijd zijn dan?

Reply:
Nee, in het geval dat er geen vl-radio-group gebruikt wordt is er geen assignedSlot aanwezig.